sql/mob_skills.sql

```diff
@@ -1169,7 +1169,7 @@ INSERT INTO `mob_skills` VALUES (1332,63,'extremely_bad_breath',4,10.0,2000,1500
 -- INSERT INTO `mob_skills` VALUES (1334,154,'contamination',0,7.0,2000,1500,4,0,0,0,0,0,0);
 -- INSERT INTO `mob_skills` VALUES (1335,151,'toxic_pick',0,7.0,2000,1500,4,0,0,0,0,0,0);
 INSERT INTO `mob_skills` VALUES (1336,289,'frenzied_rage',0,7.0,2000,1500,1,0,0,0,0,0,0);
--- INSERT INTO `mob_skills` VALUES (1337,1081,'charm',0,7.0,2000,1500,4,0,0,0,0,0,0);
+INSERT INTO `mob_skills` VALUES (1337,1081,'charm_copycat',0,7.0,2000,1500,4,0,0,0,0,0,0);
```

Comment:
Is this the only place this mobskill is used?

Reply:
Yes, as far as I could tell it was unique to Osschaart. Was also the ID being used in the capture